matscipy/calculators/manybody/potentials.py

```diff
@@ -74,3 +78,53 @@ def hessian(self, r_p, xi_p):
             np.zeros_like(xi_p),
             np.zeros_like(xi_p),
         ])
+
+class HarmonicAngle(Manybody.Theta):
+    """
+    Implementation of a harmonic angle interaction.
+    """
+
+    def __init__(self, k0=0.1, theta0=149.47):
+        self.k0 = k0
+        self.theta0 = theta0
+
+    def __call__(self, rij, rik, rjk):
```

Comment:
Should we make a decorator to convert the formulation in terms of distance to distance squared?

Reply:
I think this might make sense in order to keep the code generic